repo/packages/Q/quobyte/0/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "quobyte",
+  "version": "1.2",
+  "tags": [
+    "mesosphere",
+    "framework",
+    "quobyte",
+    "storage",
+    "filesystem",
+    "scalability",
+    "mesos",
+    "performance",
+    "bigdata"
+  ],
+  "maintainer": "silvan@quobyte.com",
+  "description": "Quobyte is a software defined distributed file system, running on commodity hardware.",
+  "framework": true,
+  "scm": "https://github.com/bar/foo.git",
+  "website": "http://www.quobyte.com/",
+  "images": {
+    "icon-small": "http://www.quobyte.com/mesos/icon-quobyte-small.png",
+    "icon-medium": "http://www.quobyte.com/mesos/icon-quobyte-medium.png",
+    "icon-large": "http://www.quobyte.com/mesos/icon-quobyte-large.png",
+    "screenshots": [
+      "http://www.quobyte.com/mesos/screen-1.png",
+      "http://www.quobyte.com/mesos/screen-2.png",
+      "http://www.quobyte.com/mesos/screen-3.png"
+    ]
+  },
+  "preInstallNotes": "Welcome to Quobyte.",
```

Comment:
There should be a message of what resources are expected.  This gives the user an opportunity prior to installation to confirm the necessary resources are available in the cluster.